sources/Cdn/Cdn.csproj

```diff
@@ -0,0 +1,21 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+    <PropertyGroup>
+        <TargetFramework>net5.0</TargetFramework>
+    </PropertyGroup>
+
+    <ItemGroup>
+        <Protobuf Include="..\..\assets\protos\*.proto" GrpcServices="Server" />
```

Comment:
I was wondering, do we need to include *all* the protos when this service reads a single one? We might be bloating the bundle with unnecessary code.